src/applications/disability-benefits/526EZ/content/confirmation-page.jsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import moment from 'moment';
+
+import siteName from '../../../../platform/brand-consolidation/site-name';
+import {
+  successMessage,
+  checkLaterMessage,
+  errorMessage,
+} from '../content/confirmation-poll';
+import { get4142Selection } from '../helpers';
+
+const privateRecordReleaseContent = (
+  <div>
+    <p>
+      <strong>
+        If you need us to get your private medical records from your doctor,
+      </strong>{' '}
+      you’ll need to fill out an Authorization to Disclose Information to the VA
+      (VA Form 21-4142).
+    </p>
+    <p>
+      <a
+        href="https://www.vba.va.gov/pubs/forms/VBA-21-4142-ARE.pdf"
+        target="_blank"
+      >
+        Download VA Form 21-4142
+      </a>
+      .
+    </p>
+    <p>Please print the form, fill it out, and send it to:</p>
+    <p className="va-address-block">
+      Department of Veterans Affairs
+      <br />
+      Claims Intake Center
+      <br />
+      PO Box 4444
+      <br />
+      Janesville, WI 53547-4444
+    </p>
+  </div>
+);
+
+const template = (props, title, content, submissionMessage) => {
```

Comment:
is this name specifically recognizable?

Reply:
Not really, but it's not being exported, so I thought it made enough sense within this file. Would `confirmationPageTemplate` or something make more sense?